subsys/dult/Kconfig

```diff
@@ -148,6 +148,58 @@ config DULT_USER
 	help
 	  Add DULT user source files.
 
+config DULT_MOTION_DETECTOR
+	bool "Enable DULT Motion detector feature"
+	help
+	  The DULT Motion detector requests the device to ring if the accessory separated
+	  from owner is moving. For more details see
+	  https://www.ietf.org/archive/id/draft-detecting-unwanted-location-trackers-01.html#name-motion-detector.
+
+if DULT_MOTION_DETECTOR
+
+config DULT_MOTION_DETECTOR_TEST_MODE
+	bool "Enable DULT Motion detector test mode"
+	help
+	  The DULT Motion detector test mode allows to configure motion detector parameters
+	  for testing purposes. Those values are defined in the DULT specification and should
+	  not be changed in the production code.
+
+config DULT_MOTION_DETECTOR_SEPARATED_UT_SAMPLING_RATE1
```

Comment:
You can also think if you want to have as many options with conditional prompts. Every prompt option typically needs to be documented in the service library and the integration guide. I think that the most impactful parameters for the user testing experience is UT timeout and backoff period as they require long waiting time.